cluster/cluster.go

```diff
@@ -64,6 +64,10 @@ func (c *Cluster) Validate() error {
 		return fmt.Errorf("spec.mysqlOpts.user cannot be root|%s|%s|%s", utils.ReplicationUser, utils.OperatorUser, utils.MetricsUser)
 	}
 
+	if c.Spec.MysqlOpts.RootHost == "127.0.0.1" {
```

Comment:
Add comment，because script in percona server image ` create user 'root@127.0.0.1' ` will fail